Http/Controllers/FreeScoutGPTController.php

```
@@ -451,4 +554,131 @@ public function checkIsEnabled(Request $request)
+                return [
+                    'context' => '',
+                    'articles' => [],
+                    'error' => $errorMsg ?? 'Error fetching or parsing the article.',
+                    'userQuery' => $userQuery
+                ];
+            }
+        }
+        $context = "";
+        if ($settings->client_data_enabled) {
+            $customerName = $request->get("customer_name");
+            $customerEmail = $request->get("customer_email");
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$customerEmail'. (undefined)

(UnusedLocalVariable)